lib/jellyfish_web/api_spec/room.ex

```diff
@@ -14,6 +14,12 @@ defmodule JellyfishWeb.ApiSpec.Room do
       description: "Room configuration",
       type: :object,
       properties: %{
+        roomID: %Schema{
```

Comment:
I wrote this in elixir SDK but because we have `webhookUrl` I would opt for `roomId`